MainModule/Client/Core/Anti.lua

```diff
@@ -116,75 +116,74 @@ return function(Vargs, GetEnv)
 		local proxyMt = getmetatable(proxyDetector)
 
 		proxyMt.__index = function()
-			Detected("kick", "Proxy methamethod 8543")
+			Detected("kick", "Proxy methamethod 0x215F")
 
 			return task.wait(2e2)
 		end
 
 		proxyMt.__newindex = function()
-			Detected("kick", "Proxy methamethod 34545")
+			Detected("kick", "Proxy methamethod 0x86F1")
 
 			return task.wait(2e2)
 		end
 
 		proxyMt.__tostring = function()
-			Detected("kick", "Proxy methamethod 789456")
+			Detected("kick", "Proxy methamethod 0xC0BD0")
 
 			return task.wait(2e2)
 		end
 
 		proxyMt.__unm = function()
-			Detected("kick", "Proxy methamethod 789456")
+			Detected("kick", "Proxy methamethod 0x10F00")
 
 			return task.wait(2e2)
 		end
 
 		proxyMt.__add = function()
-			Detected("kick", "Proxy methamethod 789456")
+			Detected("kick", "Proxy methamethod 0x60DC3")
 
 			return task.wait(2e2)
 		end
 
 		proxyMt.__sub = function()
-			Detected("kick", "Proxy methamethod 789456")
+			Detected("kick", "Proxy methamethod 0x90F5D")
 
 			return task.wait(2e2)
 		end
 
 		proxyMt.__mul = function()
-			Detected("kick", "Proxy methamethod 789456")
+			Detected("kick", "Proxy methamethod 0x19999")
 
 			return task.wait(2e2)
 		end
 
 		proxyMt.__div = function()
-			Detected("kick", "Proxy methamethod 789456")
+			Detected("kick", "Proxy methamethod 0x1D14AC")
 
 			return task.wait(2e2)
 		end
 
 		proxyMt.__mod = function()
-			Detected("kick", "Proxy methamethod 789456")
+			Detected("kick", "Proxy methamethod 0x786C64")
 
 			return task.wait(2e2)
 		end
 
 		proxyMt.__pow = function()
-			Detected("kick", "Proxy methamethod 789456")
+			Detected("kick", "Proxy methamethod 0x1D948C")
 
 			return task.wait(2e2)
 		end
 
 		proxyMt.__len = function()
-			Detected("kick", "Proxy methamethod 789456")
+			Detected("kick", "Proxy methamethod 0xBE931")
 
 			return task.wait(2e2)
```

Comment:
Always wondered, why does it return a `task.wait` that yields for a really really long time?
Wouldn't a normal `return` suffice?
(Lines 121, 127, 133, 139, 145, 151, 157, 163, 169, 175, 181)

Reply:
Its better to return something and it's also shorter

Reply:
Couldn't you just return a normal Datatype like a Boolean or a nil value?